task/prefetch-dependencies-oci-ta/0.1/README.md

```diff
@@ -18,6 +18,21 @@ https://github.com/containerbuildsystem/cachi2#basic-usage.
 |ociArtifactExpiresAfter|Expiration date for the trusted artifacts created in the OCI repository. An empty string means the artifacts do not expire.|""|false|
 |ociStorage|The OCI repository where the Trusted Artifacts are stored.||true|
 
+### Config file usage
```

Comment:
`hack/generate-ta-tasks.sh` tries to remove it from here

Reply:
I think `generate-ta-tasks` uses the generate-readme script to generate the README. (And the next time someone runs that script manually, it would remove your changes). You would need to add this to the `description` of the task, not directly to the README

Reply:
I'm not very happy about adding it into description, formatting cannot be so rich and also describing something that's defined in later section seems off.

Reply:
I also cannot add example into param description as there are tools which are using that field and doesn't parse markdown

I can probably just keep it documented in original task only

Reply:
We need readme templating :D

Reply:
Works fine for https://github.com/konflux-ci/build-definitions/blob/c22a0bb0d4bcbaebb27fd4ceb317ae71b3ddae85/task/upload-sbom-to-trustification/0.1/upload-sbom-to-trustification.yaml#L8-L30

But yes, we do need README templating

Reply:
How does it look in konflux webUI?

Reply:
You can view task descriptions in Konflux web UI? :open_mouth:

Reply:
![image](https://github.com/user-attachments/assets/78f09c02-be4c-4468-bdce-4fcef68ae049)
 it seems that it can parse markdown

Reply:
Updated, I think that this could work

Reply:
Nice